docs/source/developer/interface/command-line-parameters.md

```diff
@@ -0,0 +1,35 @@
+# Command Line Parameters
+
+You can launch an interface with these parameters.
+
+Displays help information
+
+    -h, --help
+
+Displays version information
+
+    -v, --version
+
+ICE-Server address
+
+    -i <IP:PORT or HOSTNAME:PORT>
+
+Domain Server UUID
+
+    -d <uuid>
+
+Request a temporary domain name
+
+    --get-temp-name
+
+<path> User config file path
+
+    --user-config
+
+Displays help information
```

Comment:
```suggestion
Displays the PID of the parent process
```
Or does it set the PID of the parent process, in which case the parameter is missing?